Provider/PhpcrMenuProvider.php

```diff
@@ -188,7 +189,11 @@ public function get($name, array $options = array())
      */
     public function has($name, array $options = array())
     {
-        return $this->find($name, $options, false) instanceof NodeInterface;
+        try {
+            return $this->find($name, $options, false) instanceof NodeInterface;
+        } catch (NamespaceException $e) {
```

Comment:
do you get a namespace exception when the name has a colon in it? that indeed should not break us. there could be other invalid paths i think, e.g. with a:b:c - can you try with that too and see what exception is thrown? i don't want to catch RepositoryException, as that would hide if the phpcr is down or fundamental setup mistakes.

we could also check whether $name is a valid phpcr name - that would avoid the complete roundtrip to phpcr if you have non-phpcr menues. this method from phpcr-utils could help: https://github.com/phpcr/phpcr-utils/blob/master/src/PHPCR/Util/PathHelper.php#L47